server/.env.example

```
@@ -1,4 +1,4 @@
-DATABASE_URL=
+DATABASE_URL="postgresql://postgres:1234@localhost:5432/internhack"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)